Assets/Scripts/Interactables/MountInteractable.cs

```diff
@@ -28,6 +28,8 @@ public class MountInteractable : Interactable
 
     public override Interaction InteractableInteraction { get => Interaction.MOUNT; }
 
+    public override Interactor InteractableInteractor { get => Interactor.KNIGHT; }
```

Comment:
Perhaps create a serialized field so that the `Interactor` for each interactable can be selected in the inspector, then get the value of the serialized field in the getter here. This would allow for the case where only the Dragon can interact with runes in some other future puzzle.